games/game.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../items'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_date
+
+  def initialize(name, publish_date, multiplayer, last_played_date)
+    super(name, publish_date)
+    @multiplayer = multiplayer
+    @last_played_date = last_played_date
+  end
+
+  def to_s
```

Comment:
this method is not in the requirements, you can delete it

Reply:
this one was to make sure that everything is handled as a string so it will ease my work later, but i will remove it anyway